airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -802,20 +1281,33 @@ definitions:
         type: string
         enum: [ListPartitionRouter]
       cursor_field:
+        title: Cursor Field
+        description: While iterating over list values, the name of field used to reference a list value.
```

Comment:
Can we mention here how to access it somewhere else?

Reply:
done